### DISCUSSION TITLE
Menu - setting width on popover

### DISCUSSION THREAD

**Discussion by binhxn** [ariakit/ariakit#1974] [Questions] [answered]
Hello 👋 ,

I have a use case where I need to make a responsive `Menu` component. That is, displaying a full-width Menu popover for mobile breakpoints. I tried adding inline styles, such as `style={{ width: '100%' }}`, but it seems like the layout is controlled by the parent popover element and I can't seem to access it. Any ideas on how to approach this?

The screenshot below points to the parent popover element that I believe I need to handle width styles on.

![CleanShot 2022-10-17 at 14 51 26](https://user-images.githubusercontent.com/8155387/196290789-04afee08-84c0-420a-a57c-a0680f37a882.png)

**Comment by diegohaz** [comment, marked as answer]
https://github.com/ariakit/ariakit/discussions/1905#discussioncomment-3763432
